listenbrainz/listenstore/timescale_listenstore.py

```diff
@@ -34,8 +34,17 @@
 NUMBER_OF_USERS_PER_DIRECTORY = 1000
 DUMP_FILE_SIZE_LIMIT = 1024 * 1024 * 1024  # 1 GB
 DATA_START_YEAR = 2005
-SECONDS_IN_TIME_RANGE = 432000
+DATA_START_YEAR_IN_SECONDS = 1104537600
 
+# This value MUST match the time_bucket definition of the listen_count view.
+# See admin/timescale/create_views.sql for more details.
+LISTEN_COUNT_BUCKET_WIDTH = 86400 * 5
+
+# How many listens to fetch on the first attempt. If we don't fetch enough, doublt it, try again.
```

Comment:
"doublt it"
typo aside, if I understand correctly we're multiplying by WINDOW_SIZE_MULTIPLIER rather than doubling.